src/pages/cookie-policy/CookiePolicy.js

```diff
@@ -0,0 +1,33 @@
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { Box, Container } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import { styles } from './cookie-policy.styles'
+import { cookieItemsData } from './cookie-policy.constants'
+
+const CookiePolicy = () => {
+  const { t } = useTranslation()
+
+  const itemsData = cookieItemsData.map((item) => {
+    return (
+      <Box data-testid="cookieItems" key={ item.id } sx={ styles.sectionTitle }>
```

Comment:
If the first element needs to have other CSS styles, you should use CSS for that.

Reply:
fixed